libpod/stats_linux.go

```diff
@@ -43,6 +43,23 @@ func (c *Container) getPlatformContainerStats(stats *define.ContainerStats, prev
 	if err != nil {
 		return err
 	}
+	var newNetStats map[string]define.ContainerNetworkStats
+	if len(netStats) > 1 {
+		newNetStats = make(map[string]define.ContainerNetworkStats)
+		for net, stats := range netStats {
+			newStats := define.ContainerNetworkStats{
+				RxBytes:   stats.RxBytes,
+				RxDropped: stats.RxDropped,
+				RxErrors:  stats.RxErrors,
+				RxPackets: stats.RxPackets,
+				TxBytes:   stats.TxBytes,
+				TxDropped: stats.TxDropped,
+				TxErrors:  stats.TxErrors,
+				TxPackets: stats.TxPackets,
+			}
+			newNetStats[net] = newStats
+		}
```

Comment:
Instead of this here I would really just make getContainerNetIO(c) the right type ContainerNetworkStats already do avoid looping here again.